main/webapp/modules/core/langs/translation-en.json

```diff
@@ -205,6 +205,8 @@
     "core-dialogs/reorder-column": "Re-order / Remove columns",
     "core-dialogs/drag-column": "Drag columns to re-order",
     "core-dialogs/drop-column": "Drop columns here to remove",
+    "core-dialogs/add-all": "Add All",
+    "core-dialogs/remove-all": "Remove All",
```

Comment:
To keep the capitalization consistent with the rest of the tool, those should be `Add all` and `Remove all` (second word not capitalized).